docs/node/migrations/zero-downtime-migrations/tutorial.mdx

```diff
@@ -0,0 +1,150 @@
+kwil-admin migrate genesis-state --out-dir /tmp/chain2
+```
+
+Note that the command will not download any files if there is no approved migration or the migration has not yet started. Please retry the command after the migration is activated.
+
+```text
+No genesis state to download yet. Migration is set to start at block height: 143
+```
+
+:::warning
+The `genesis.json` and `snapshot.sql.gz` files are generated in the directory specified using the `--out-dir` flag. Ensure that you don't run the command with `--root-dir` flag set to the current node's root directory as it can overwrite the existing genesis file in the node's root directory.
+
+If the admin service is directly accessible from the new node (e.g. it is listening on a TCP socket securely accessible from the new node) it can be run on the new node directly.
+:::
+
+```bash
+kwil-admin migrate genesis-state --root-dir /tmp/chain2
+Migration State:
+        Start Height: 143
+        End Height: 2143
+        Downloaded Genesis File: .testnet2/genesis.json
+        Downloaded Snapshot File: .testnet2/snapshot.sql.gz
+```
+
+:::info
+If the node is still catching up, the genesis file won't be generated until it has fully synced with the network.
+:::
+
+
+## Start New Network
+The new network should be configured to bootstrap from the snapshot file provided by the old network and replicate the state changes from the old network during the migration period. 
+
+1.  Install the new versions of the `kwild` binaries. Please refer to the [installation guide](/docs/daemon/installation) for more details.
+2. [Download the snapshot and genesis files](#genesis-state) generated at the start of the migration as shown above.
+    The genesis file downloaded will have the following information set:
+    - `chain_id`: This is set based on the chainID in the migration proposal.
+    - `app_hash`: This field is set based on the hash of the downloaded snapshot file at the migration start height.This should not be changed as it must match the genesis state snapshot.
+    - `validators`: This field is set to the validators from the existing network.
+    - `consensus_params.migration.start_height`: The start height of the migration on the old network. Specifies the height from which the state changes are to be replicated onto the new network.  This should not be changed as it must match the migration proposal on the old network.
+    - `consensus_params.migration.end_height`: The end height of the migration on the old network. Specifies the height until which the state changes are to be replicated onto the new network.  This should not be changed as it must match the migration proposal on the old network.
+
+    :::warning
+    `allocs` and `activations` fields are set to `null` in the genesis file. All other fields have same configurations as that on the old network. If any other changes are needed to the genesis configuration, update the genesis file manually and redistribute it to the node operators.
```

Comment:
Do account balances migrate? Why is this dangerous?

Reply:
updated.